administration.html

```diff
@@ -0,0 +1,272 @@
+        </a>
+
+        <nav>
+            <ul class="nav__links">
+                <li class="nav__item"><a href="https://api.gouv.fr/apropos">À propos</a></li>
+                <li class="nav__item"><a
+                        href="mailto:contact@particulier.api.gouv.fr?subject=Contact%20via%20particulier.api.gouv.fr">Nous
+                    contacter</a></li>
+            </ul>
+        </nav>
+    </div>
+</header>
+
+<div class="hero" role="banner">
+    <div class="hero__container administration">
+        <div class="row">
+            <div class="column ">
+                <h1>Pas de file d'attente dans ma mairie</h1>
+                <p>À chaque rentrée vos usagers doivent faire la queue pour faire
+                    calculer leurs aides, certains posent même une demi-journée pour faire ces démarches.</p>
+                <p>La bonne nouvelle ?
+                    <a style="color: #1631d5;" href="https://particulier.api.gouv.fr/FAQ.html">API Particulier résout ce
+                        problème</a>.
+                    <b> API Particulier est un service gratuit fourni par la <abbr
+                            title="Direction interministérielle du numérique et du système d'information et de communication de l'État">DINSIC</abbr>.</b>
+                </p>
+                <div>
+                    <form id="gform"
+                          action="https://script.google.com/macros/s/AKfycbzSuEezvl9QSpGbtOsP_F__g_A-vua8KgWrIK9K/exec"
+                          method="post" style="margin: 0">
+                        <div id="form-elements">
+                            <div style="margin-top: 0.5em">
+                                <input id="gform-input-name" type="text" placeholder="Mon administration (ex : Maire de Clamart)" name="name">
+                            </div>
+                            <div style="margin-top: 0.5em">
+                                <input type="email" placeholder="Mon adresse email (ex : claude@ville-clamart.fr)"
+                                       name="email">
+                            </div>
+                            <div class="honeypot">
+                                <input type="text" name="honeypot">
+                            </div>
+                            <div style="margin-top: 1em">
+                                <button class="button" type="submit">Faire un test pour mon administration</button>
+                            </div>
+                        </div>
+                        <div id="thank-you-message" class="notification success" style="display: none">Merci ! Nous
+                            revenons
+                            vers vous au plus vite.
+                        </div>
+                    </form>
+                </div>
+            </div>
+            <div class="column illustration">
+                <img src="img/file-d-attente.png" alt="File d'attente">
```

Comment:
I'm nos a huge fan of the illustration. Is this the final design ?

Reply:
no ! it's a draft but I found this better than a placeholder